ckanext/ytp/comments/email_notifications.py

```diff
@@ -151,30 +152,7 @@ def get_admins(owner_org, user, content_type, content_item_id):
     return users
 
 
-def notify_admins(owner_org, user, template, content_type, content_item_id, comment_id):
-    """
-
-    :param owner_org: organization.id of the content item owner
```

Comment:
why comment deleted?

Reply:
Hi @duttonw, while implementing this story https://salsadigital.atlassian.net/browse/DQL2-28, the solution direction mentioned "Adjust the notify_admins and notify_admins_and_comment_notification_recipients functions in helpers.py of the ckanext-ytp-comments extension" however I noticed the method 'notify_admins' was not referenced anywhere in the code, so I decided to remove the method. Is there a reason you want me to re-add it?

Reply:
is good to prune unused code. 👍

Reply:
actually thinking about it, we should leave this in and mark it as deprecated as others may be referencing this as this is a plugin that more than us use. (if we want to contribute back to the main repo)

Reply:
Yeh that could be true, will add it back in.

Reply:
@duttonw Re-added 'notify_admins' method with a deprecation decorator